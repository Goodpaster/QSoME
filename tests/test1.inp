#Geometries taken from https://pubs.acs.org/doi/suppl/10.1021/ja4102979/suppl_file/ja4102979_si_001.pdf
subsystem
O 0. 0. 0.000
hl_method_num 1
end

subsystem
H  0. -0.757 0.587
H  0.  0.757 0.587
end

basis
 default def2-svp
end

env_method_settings
 env_method b3lyp
 embed_settings
  max_cycle 50
  diis_num 2
 end
end

hl_method_settings
    hl_order 1
    hl_method rhf
end

opt_geom_settings
    max_cycle 100
end

