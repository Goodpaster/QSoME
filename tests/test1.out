iter: 89:0               |ddm|:7.123848e-09               |Tr[DP]|:3.693286e-01
iter: 89:1               |ddm|:8.642967e-10               |Tr[DP]|:3.693286e-01

SUBSYS E
-1256.6841105341189
EMB E
11.330869391390687
PROJ E
0.36932856736790054
Subsystem 0 Energy:                                               -1244.98391258
SUBSYS E
-746.8745252025722
EMB E
-45.63476884553545
PROJ E
0.3693285673465232
Subsystem 1 Energy:                                                -496.22901257

********************************************************************************
TIMING: Freeze and Thaw                                               1169.6412s
KS-DFT  Energy:                                                   -1709.61811657
TIMING: Supersystem Energy                                               0.0000s
********************************************************************************
                             Env-in-Env Calculation
********************************************************************************
Env-in-Env Energy:                                                -1708.63146560
********************************************************************************
TIMING: Env. in Env. Energy                                              0.0026s
********************************************************************************
                           HL Subsystems Calculation
********************************************************************************
Overwritten attributes  get_fock energy_elec  of <class 'pyscf.scf.uhf.UHF'>
converged SCF energy = -1248.42737729225  <S^2> = 3.7536266  2S+1 = 4.0018129
